_data/food.yml

```
@@ -34,10 +34,18 @@ websites:
       tfa: No
       twitter: tacobell
       facebook: tacobell
+  
+    - name: Wolt
+      url: https://wolt.com/
+      img: wolt.png
+      tfa: No
+      twitter: woltapp
+      facebook: woltapp
+      email_address: info@wolt.com
  
     - name: Zomato
       url: https://www.zomato.com
       img: zomato.png
       tfa: No
       twitter: Zomato
-      facebook: zomato
+      facebook: zomato
```

All files should end with a blank line.
I recommend you to use a decent text editor like Atom, Notepad++, VS Code, Sublime etc. They fix that for you.
```suggestion
      facebook: zomato

```